pom.xml

```diff
@@ -97,6 +97,11 @@
       <artifactId>org.everit.json.schema</artifactId>
       <version>1.11.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.neovisionaries</groupId>
+      <artifactId>nv-i18n</artifactId>
```

Comment:
Is there a reason we need this external dependency vs. using the Java provided `Locale` class? It seems like it would be better to rely on Java keeping the ISO 3166-1 list up to date than an external project.

Reply:
I chose this dependency because it can be updated independently of jre.
But it is possible to use Locale class if there are concerns about using external dependency